docs/install.rst

```diff
@@ -116,7 +118,7 @@ sub-repos. To download them in this manner, simply run:
 
 .. code:: bash
 
-    git submodule update --init
+    git submodule update --init --recursive
```

Comment:
Would it be worth mentioning that we can also clone pavilion and the submodules from start with: `clone --recursive https://github.com/hpc/pavilion2`?

Reply:
This is fixed elsewhere. 

I was thinking of putting it in a git post-update hook or something.